crates/api_crud/src/post/create.rs

```diff
@@ -62,6 +66,22 @@ impl PerformCrud for CreatePost {
     check_community_ban(local_user_view.person.id, data.community_id, context.pool()).await?;
     check_community_deleted_or_removed(data.community_id, context.pool()).await?;
 
+    let community_id = data.community_id;
+    let community = blocking(context.pool(), move |conn| {
+      Community::read(conn, community_id)
+    })
+    .await??;
+    if community.posting_restricted {
+      let community_id = data.community_id;
+      let is_mod = blocking(context.pool(), move |conn| {
+        CommunityView::is_mod(conn, local_user_view.local_user.person_id, community_id)
```

Comment:
Admins should also be able to post to these communities, so "is_mod_or_admin". I could see this being necessary to give warnings, and also admins shouldn't really be restricted from doing anything except editing other people's content.

Reply:
alright, done